src/config.js

```diff
@@ -27,6 +27,7 @@ const baseConfig = {
       host: 'https://master.pfe-preview.zooniverse.org/',
       projectId: '1764',
       projectSlug: 'wgranger-test/anti-slavery-testing',
+      workflowId: '3017'
```

Comment:
PR #103 also adds in [Workflow ID](https://github.com/zooniverse/liberating-the-liberator/pull/103/files#diff-9d9a6cd82f41984872a66a3ab0d440c4R31) so a note has to be made during a PR rebase (depending if either 105 or 103 is merged first).